src/snowflake/snowpark/session.py

```diff
@@ -274,6 +274,7 @@ def __init__(self, conn: ServerConnection) -> None:
         self._file = FileOperation(self)
 
         self._analyzer = Analyzer(self)
+        self.sql_simplifier_enabled = False
```

Comment:
Will it always be `false`? Should we not read the if the env variable is set and enable simplifier accordingly?

Reply:
Default to be false. Users can change the value to opt in. The environment variable is only in our test code.